examples/dataflow-streaming-benchmark/src/test/java/com/google/cloud/pso/pipeline/StreamingBenchmarkTest.java

```diff
@@ -114,12 +157,12 @@ public void testMessageGeneratorInvalidSchema() throws IOException {
 
     // Assert
     //
-    PAssert.that(results).satisfies(input -> {
-      PubsubMessage message = input.iterator().next();
-      assertThat(message, is(notNullValue()));
-      assertThat(new String(message.getPayload()), is(equalTo(schema)));
-      return null;
-    });
+    //    PAssert.that(results).satisfies(input -> {
+    //      PubsubMessage message = input.iterator().next();
+    //      assertThat(message, is(notNullValue()));
+    //      assertThat(new String(message.getPayload()), is(equalTo(schema)));
+    //      return null;
+    //    });
```

Comment:
Why comment out the assertion?

Reply:
With latest code the invalid schema would throw an exception and the same was asserted inside Test Annotation.  Also removing the commented code completely to avoid confusion.